prosody/Dockerfile

```diff
@@ -31,7 +25,18 @@ RUN \
       libssl1.0-dev \
       liblua5.2-dev \
     && apt-cleanup \
-    && rm -rf /etc/prosody
+    && rm -rf /etc/prosody /var/cache/apt
```

Comment:
any reason to duplicate the cleanup steps? we could run them once, at the end

Reply:
Because every layer is persistent. So for get less image we have to clean unused files in which layer.

Reply:
Ah, true, I always forget.